MUXSDKStats/MUXSDKStats.xcodeproj/project.pbxproj

```diff
@@ -744,7 +743,7 @@
 				BUILD_LIBRARY_FOR_DISTRIBUTION = NO;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = GFDDVAVUVC;
```

Comment:
This is my personal development team which shouldn't be in the project. I changed tour project to not codesign because the application developer will codesign their project with their own team anyway.
![Screen Shot 2021-04-12 at 1 20 54 PM](https://user-images.githubusercontent.com/1444681/114435256-f8b35e80-9b91-11eb-9e6d-b976bb7ae3dd.png)